dev-docs/RFCs/v8.0/ui-module-rfc.md

```diff
@@ -0,0 +1,137 @@
+# UI Module RFC
+
+* **Author**: Xiaoji Chen
+* **Date**: December 2022
+* **Status**: **Pending Review**
+
+## Overview
+
+Visualization applications often require some chrome around the WebGL canvas to offer better user experience. While it is possible for every app to implement their own UI components that interact with deck.gl, it is no trivial effort to implement them well. 
+
+This RFC proposes to add a new module that packages common UI components that seamlessly integrate with deck.gl, including but not limited to:
+
+- Marker: HTML element pinned to a location in the world space
+- Navigation: buttons to zoom / rotate / reset the viewport
```

Comment:
Each view controller type could have their own. 

A gimble widget for OrbitView's orientation.

A mini globe for MapView or GlobeView.

Could it make sense to implement minimaps as a UI controller, since their often for navigation?